test/dump/try-details.txt

```diff
@@ -14,16 +14,21 @@
   )
 )
 (;; STDOUT ;;;
+
 try-details.wasm:	file format wasm 0x1
+
 Section Details:
+
 Type:
  - type[0] () -> i32
 Function:
  - func[0] sig=0
 Custom:
  - name: "exception"
  - except[0] (i32)
+
 Code Disassembly:
+
```

Comment:
Yeah, I think they accidentally got included when I rebuilt all the tests. It is that same issue you found before, where empty lines are not caught by the diff.